packages/lesswrong/components/common/SeparatorBullet.tsx

```diff
@@ -3,8 +3,9 @@ import { registerComponent } from '../../lib/vulcan-lib';
 
 const styles = (theme: ThemeType): JssStyles => ({
   root: {
-    marginLeft: 3,
-    marginRight: 3
+    marginLeft: 10,
```

Comment:
Are you sure this is right? Aren't we using SeparatorBullets all over the place? Have we checked this doesn't break any of the existing ones?

Reply:
We're using the SeparatorBullet component in... zero times, and using the styles a couple times, and each time it's in a flexBox that automatically adds more than 3 pixels.